client/components/Common/Icon.stories.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import styled from 'styled-components'
```

Comment:
nits:
Please remove unused import.
Sorry, I have should add a `import/no-unused-modules` rule to ESLint.

```suggestion
```

Reply:
Sorry, I forgot removing this line…